Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TwoHandManipulatable.cs

```diff
@@ -255,19 +244,19 @@ private void UpdateStateMachine()
                     {
                         switch (ManipulationMode)
                         {
-                            case TwoHandedManipulation.Scale:
+                            case TwoHandManipulatableMode.Scale:
                                 newState = State.Scaling;
                                 break;
-                            case TwoHandedManipulation.Rotate:
+                            case TwoHandManipulatableMode.Rotate:
                                 newState = State.Rotating;
                                 break;
-                            case TwoHandedManipulation.MoveScale:
+                            case TwoHandManipulatableMode.MoveScale:
                                 newState = State.MovingScaling;
                                 break;
-                            case TwoHandedManipulation.RotateScale:
+                            case TwoHandManipulatableMode.RotateScale:
                                 newState = State.RotatingScaling;
                                 break;
-                            case TwoHandedManipulation.MoveRotateScale:
+                            case TwoHandManipulatableMode.MoveRotateScale:
                                 newState = State.MovingRotatingScaling;
```

Comment:
Why doesn't state and TwoHandManipulatableMode use the same enum?

Reply:
If they did you could shorten this whole switch statement to just be:
```
newState = TwoHandManipulatableMode;
```

Reply:
@StephenHodgson I would recommend a separate PR for this change since I don't know the details about states handling cases. The scope of this PR is making the manipulation mode as public.